eve_sqlalchemy/config/fieldconfig.py

```diff
@@ -56,11 +56,20 @@ def _render(self):
     def _get_field_nullable(self):
         return getattr(self._sqla_column, 'nullable', True)
 
+    def _has_field_server_default(self):
+        check = getattr(self._sqla_column, 'server_default')
+        if check:
+            return True
+        else:
+            return False
+
```

Comment:
This can be done simpler like this:
```python
return bool(getattr(self._sqla_column, 'server_default')
```

Reply:
Personally I would rename that method to just `has_server_default` as 'field' refers to the entry in Eve's resource configuration, not to the SQLAlchemy column.